.travis.yml

```diff
@@ -1,12 +1,14 @@
 language: java
 jdk:
-- oraclejdk8
+  - oraclejdk8
+  - oraclejdk11
 
 dist: trusty
 sudo: required
 
 install: true
 
+before_script: "./.travis/set-jdk8-settings-if-possible.sh"
```

Comment:
I'd rather run a separate jdk11 build (after everything else has run) in the same plan:

```
  - stage: build
    script:
    - mvn --batch-mode -q -Djava.version=11 -Pskip-it clean verify
```

The script could possibly change the JAVA_HOME to the Java 11 JDK (while the others run with Java 8). But I don't really have experience with Travis and can't tell how this should be done here.

Reply:
It is possible indeed. They allow you to switch jdks with a provided script. I'll have a look at it.

Reply:
Done in 6bd41c1deb2a348d4f6dcee2f6e1f48c0376b6d3. See the [build jobs](https://github.com/bbortt/citrus-simulator/blob/feature/prepare-maven-build-for-openjdk11/.travis.yml).